### ISSUE TITLE
12gb of Vram and got COOM error

### ISSUE THREAD

**Issue by slimanibenbadis** [pharmapsychotic/clip-interrogator-ext#27]
Hi, the extension was working perfectly before, and since the recent update I got this error
**`CUDA out of memory. Tried to allocate 920.00 MiB (GPU 0; 12.00 GiB total capacity; 9.16 GiB already allocated; 0 bytes free; 10.68 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF`**

It is very surprising because I have a 3060 card with its 12gb of VRAM.

**Comment by SanAnt0n10** [comment]
Exact same problem here. Default settings, using the interrogator right after starting the UI.
It happens during the Flavor Chain step.
With blip-base, the VRAM used reaches 11.7 GB.

Loading CLIP Interrogator 0.6.0...
Loading caption model blip-large...
Loading CLIP model ViT-L-14/openai...
Loaded CLIP model and data in 7.47 seconds.
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 55/55 [00:00<00:00, 219.75it/s]
Flavor chain: 0%| | 0/32 [00:01<?, ?it/s]
CUDA out of memory. Tried to allocate 918.00 MiB (GPU 0; 12.00 GiB total capacity; 8.99 GiB already allocated; 0 bytes free; 10.63 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation. See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF

**Comment by slimanibenbadis** [comment]
Yes, I also forgot to mention that the error strike even with blip-base which is supposed to use less VRAM.

I don't know what to do, it is very annoying because I used the extension for captioning of datasets destined to train some LORAs. I found it better for this than simple BLIP captioning.

**Comment by lavalava45** [comment]
Oh my Gosh. Good that I'm not alone with that issue. Already reinstalled everything cause could not find similar issues here. Also started to get CUDA out of memory errors on my 3060 12Gb card. Also even if the process interrogation starts (after the clear restart and killing every other app) it takes forever stooping at the "Flavor chain" at 18-19%.

**Comment by pharmapsychotic** [comment]
Sorry I thought the 12GB GPUs would be able to handle the high VRAM settings. I've switched it so 12GB uses lower VRAM config now and should get you up and running again.

**Comment by SanAnt0n10** [comment]
Thank you! Now it also works with blip-large.
Using blip-large now reaches only 8.8GB, and blip-base not much less.
But the results are different so I suppose it really uses different models.
